e2e/testdata/test-complex-service/main.go

```diff
@@ -10,38 +10,23 @@ import (
 	"time"
 
 	"github.com/mesg-foundation/engine/ext/xsignal"
-	"github.com/mesg-foundation/engine/hash"
-	pb "github.com/mesg-foundation/engine/protobuf/api"
+	"github.com/mesg-foundation/engine/server/grpc/runner"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/keepalive"
 )
 
 const (
 	// env variables for configure mesg client.
-	envMesgEndpoint     = "MESG_ENDPOINT"
-	envMesgInstanceHash = "MESG_INSTANCE_HASH"
+	envMesgEndpoint  = "MESG_ENDPOINT"
+	envMesgMsg       = "MESG_MSG"
+	envMesgSignature = "MESG_SIGNATURE"
```

Comment:
what do you think to rename this envs to `MESG_REGISTER_MSG` and `MESG_REGISTER_SIGNATURE`?

Reply:
the 2 envs has been replaced by `MESG_REGISTER_PAYLOAD`